### PR TITLE
csv export options need to be HashWithIndifferentAccess

### PR DESCRIPTION
**Pull request by loicginoux**
a correction for this bug.
https://github.com/sferik/rails_admin/issues/2740

### CODE REVIEW

File: lib/rails_admin/support/csv_converter.rb [hunk 1]
```diff
@@ -32,6 +32,7 @@ def initialize(objects = [], schema = {})
     end
 
     def to_csv(options = {})
+      options = HashWithIndifferentAccess.new(options)
```
Comment (mshibuya) [inline lib/rails_admin/support/csv_converter.rb]: Calling [`symbolize_keys!`](http://api.rubyonrails.org/classes/Hash.html#method-i-symbolize_keys-21) would be a bit simpler, I guess.

**Comment by mshibuya** [comment]
Test code needed.

File: lib/rails_admin/support/csv_converter.rb (hunk 1 above)
Comment (loicginoux) [inline lib/rails_admin/support/csv_converter.rb]: I guess this is easier but I prefered to use the safest option. First, not knowing the rest of the code and the consequences, I prefered to be able to use string and symbol as keys, to be safe. Also, you suggestion would actually be to use `.deep_symbolize_keys!` 
At least using `HashWithIndifferentAccess`, I am sure I am not introducing any regression.

**Comment by mshibuya** [comment]
Can be merged when rubocop offenses are resolved.
https://travis-ci.org/sferik/rails_admin/jobs/174276072#L425-L430

**Comment by loicginoux** [comment]
done.

**Comment by mshibuya** [comment]
Thank you!

**Pull request merged**